src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+
+Head Revolutionaries share the same team and when they are given the role they obtain the names and jobs of the other Head Revolutionaries on the station. Revolutionaries can be identified by other revolutionaries via an icon next to their character, however Head Revs do not have a unique icon.
+
+For crew, the goal is to deconvert revolutionary converts, protect Heads from being converted, and ensure Head Revs are discovered and/or do not make it onto the evacuation shuttle. 
+
+### Win Conditions
+
+Win conditions for a round with a Revolutionary antag are the following:
+
+- Revolutionary Major Victory: No *alive, unrestrained* Command *on the evac shuttle* are non-rev, >**50%** of Command are converted Revolutionaries.
+- Revolutionary Minor Victory:  No *alive, unrestrained* Command *on the evac shuttle* are non-rev, <**50%** of Command are converted Revolutionaries.
+- Rev-Crew Tie: Both Rev Minor & Crew Minor criteria are met, or neither Rev Minor & Crew Minor criteria are met.
+- Crew Minor Victory: No *alive, unrestrained* headrevs are *on the evac shuttle* when Evac reaches CentComm.
+- Crew Major Victory: *All* revolutionaries are *deconverted* or *dead* when Evac reaches CentComm.
+
+The round-end screen could look like the following:
+
+```
+Revolutionary Major Victory! [Green text]
+X% of Crew were converted to the Revolution. [Yellow text]
+X% of Command were converted to the Revolution.  [Green text]
+0 Unconverted Command made it to CentComm. [Green text]
+X Head Revolutionaries made it to CentComm. [Green text if >0]
+```
+
+```
+Revolutionary Minor Victory! [Green text]
+X% of Crew were converted to the Revolution. [Yellow text]
+X% of Command were converted to the Revolution. [Red text]
+0 Unconverted Command made it to CentComm. [Green text]
+X Head Revolutionaries made it to CentComm. [Green text if >0]
+```
+
+```
+Revolutionary-Crew Tie! [Green text]
+X% of Crew were converted to the Revolution. [Yellow text]
+X% of Command were converted to the Revolution. [Red text]
+0 Unconverted Command made it to CentComm. [Red text]
+0 Head Revolutionaries made it to CentComm. [Red text]
+```
+
+```
+Crew Minor Victory! [Green text]
+X% of Crew were converted to the Revolution. [Green text]
+X% of Command were converted to the Revolution. [Green text]
+X Loyal Command made it to CentComm. [Red text]
+0 Head Revolutionaries made it to CentComm. [Red text]
+```
+
+```
+Crew Major Victory! [Green text]
+All Revolutionaries were killed or deconverted! [Red text]
+```
```

Comment:
Overall: the goals need reworking. A conversion antag is a "crew versus the bad guys" mode, just like nukies. Escaping on the shuttle at the end is the luxury of a loyalist crew (or a fleeing loyalist crew giving up the station to the mutineers). It makes more sense if the Revs are trying to claim the station, and thus they have to _remain_ on the station when the shuttle leaves, having killed the leadership they need to claim the station for... like, the Syndicate, but shhhh.

In general you can't just have the HRs have their greentext tied to the shuttle arrival at Centcomm moment, because:

1. It makes no sense. You overthrow the heads and then... go back to NT Centcomm?
2. It encourages gameplay centred on evac and the shuttle. The HRs need to kill or restrain the heads, the heads they need to kill or restrain are always going to be on the shuttle, so just bomb the shuttle. But because it's a Revs round rather than a Tot round, the station has no tots to actually cause chaos. That's a recipe for a long, boring round that ends with a shuttle bombing, which is everyones' least favourite SS14 rounds.

Reply:
I just entirely disagree with this point because it is working under the previous Rev framing. Yes, the goals don't make sense if you perceive the intent of Revs to be "take over the station and free it from CC's clutches". But that is only if you choose that framing. 

"Start the revolution by conquering the station, and then head to CentCom to spread the revolutionary spirit to new stations! Don't let the NT loyalist scum get back home!" Put something like that in the antag description pop-up and you've made it clear that the goal revolves around eventually leaving the station.